### DISCUSSION TITLE
How to capture non-error events

### DISCUSSION THREAD

**Discussion by kwybranowski** [getsentry/sentry-react-native#3655] [General]
Hello,

I'm looking for some clarity on the best practices for capturing non-error events in Sentry. We are currently using Sentry to capture our error events - which show up in the Issues tab of the Sentry dashboard (as expected) and in the list of events in the Discover tab. 
![Screenshot 2024-03-05 at 1 13 51 PM](https://github.com/getsentry/sentry-react-native/assets/160867238/6255bea4-b0f1-4fb9-9df6-4bbcf520198f)

But, in the cases where we don't have an error but want to capture breadcrumbs and specific events (ex: Starting an EV charging session, including the logs leading up to this event), we are using the `Sentry.captureMessage` function. According to [this documentation](https://docs.sentry.io/platforms/javascript/usage/#capturing-messages), `captureMessage` seems like the right tool for this job. However, in practice, our messages end up mixed in with our error events in the Issues and Discover tabs. Is this correct? It feels like non-error messages should be tracked (and further communicated in reports/email alerts) in a way that's differentiated and less critical than how errors are.

For context: We are building an Electric Aircraft charging (& beyond!) mobile app.

Thanks for your help!

**Comment by MindStudioOfficial** [comment]
I was wondering the same thing. 

I am using captureEvent with a SentryLevel.info but it still shows up as an error and is treated as such.
Is there a better way of doing this or is Sentry not the right tool for this use case?

**Comment by Mikozans** [comment]
Same here.

We've even reached the quota in just a few days because of this.

**Comment by MattAgn** [comment]
I think you can use breadcrumbs like so https://docs.sentry.io/platforms/react-native/enriching-events/breadcrumbs/
